### ISSUE TITLE
Shorten and standardise struct names

### ISSUE THREAD

**Issue by seabbs** [CDCgov/Rt-without-renewal#499]
As discussed in #496 many of the struct names are quite long and are not 100% consistent. We should enforce consistency and also aim for less verbose struct names. 

I think an easy change is to go from

- Observation -> Obs

Any other suggestions? 

To minimise breaking changes I think we should do a single renaming PR so lets collect suggestions here.

**Comment by SamuelBrand1** [comment]
Could we not do this with some alias names?

**Comment by SamuelBrand1** [comment]
e.g. 
```julia
const NiceAlias = VerboseStructName
```

This is a cost free abstraction

**Comment by seabbs** [comment]
Yes we could and I agree that would be good for another issue (i.e add alieas for example making `AutoRegressive` exist and `AR`, `RandomWalk` and `RW` but the key issue we have here is making sure we have a standard and following it. 

Thee big thing that stands out to me is the the module is called `EpiObsModels` and we have `RecordExpectedObs` but everything else uses `Observations`

**Comment by seabbs** [comment]
(could you make the alias issue :). )
